kubernetes/ingress/ingress.go

```diff
@@ -0,0 +1,11 @@
+package ingress
+
+type Ingress struct {
```

Comment:
This would make a lot more sense to me if it was `IngressController`, not `Ingress`.